static/src/images/global/market-up.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="15" height="11" viewBox="0 0 15 11"><path fill="#298422" d="M0 9.97L1.03 11H14l1-1L8 0h-.958L0 9.984z"/></svg>
```

Comment:
Maybe, also, it would be a good idea to run them through svgo with ``- convertPathData: floatPrecision: 1`` to optimise them to one decimal place? If they survive unscathed, we could save a bit. Or two.